packages/manager/apps/dedicated/client/app/ip/components/list/list.constant.js

```diff
@@ -74,6 +74,8 @@ export const IP_COMPONENTS_LIST_TRACKING_HIT = {
 export const PAGE_SIZE_MIN = 10;
 export const PAGE_SIZE_MAX = 100;
 
+export const POLLING_INTERVAL = 10000;
```

Comment:
Is less interval possible ?

Reply:
10s is requested by the requester of this feat